.github/workflows/ci.yml

```diff
@@ -21,6 +19,37 @@ jobs:
       - name: restore build & cargo cache
         uses: Swatinem/rust-cache@v1
 
+      - name: Build
+        run: cargo build --workspace --locked
+
+      - name: compile test binaries
+        # source:
+        # https://github.com/rust-lang/cargo/issues/1924#issuecomment-289770287
+        run: "
+          mkdir ./test-binaries/ &&
+          cargo test --no-run --message-format=json |
+          jq -r \"select(.profile.test == true) | .filenames[]\" |
+          head -n 1 |
```

Comment:
Why use `head -n 1`? Right now we only have 1 test binary, but we could add integration tests in the future.

Oh, I guess it's because you hard-code the name to `docs-rs-tests`. Rather than doing that, can you just run every binary that's in the folder?

Reply:
I'll check if there is an easy way to do that. 

the jq output is giving us a second 200 MB file which we don't need.

Reply:
Oh is it really? What's the second file?

Reply:
the output is: 
```
[...]/target/debug/deps/docs_rs-e6f8c1d05a2a6574
[...]/target/debug/deps/docs_rs-e6f8c1d05a2a6574.dSYM
```

the second is a folder which contains: `Contents/Resources/DWARF/docs_rs-e6f8c1d05a2a6574`

Reply:
ah interesting ok, that must be a MacOS thing. Those don't show up on Linux, I guess because the debug info is part of the binary itself.

I wonder if it makes sense to disable debuginfo for tests? That should speed up compile times; I guess it will make backtraces less useful though.

Reply:
IMHO compile time is ok-ish, and having backtraces when running tests is quite helpful :)